course_gather/settings/prod_settings.py

```diff
@@ -6,7 +6,7 @@
 ALLOWED_HOSTS = ['*']
 
 INSTALLED_APPS.insert(0, 'whitenoise.runserver_nostatic')  # noqa: 405
-MIDDLEWARE.insert(0, 'whitenoise.middleware.WhiteNoiseMiddleware')  # noqa: 405
+MIDDLEWARE.insert(2, 'django.middleware.gzip.GZipMiddleware')  # noqa: 405
```

Comment:
I don't really see any specific reason not to have this middleware be inserted in root settings, all environments will benefit from compressed API responses. Further, I think whitenoise should be placed back here since the dev and local_dev instructions do not use it. Note, GZIP middleware needs to go above whitenoise's middleware since whitenoise reads from the response body.

Reply:
👍 I had removed Whitenoise from `prod_settings` since it was already included in `base`. Fixed in a80cf16.

Reply:
Ah oops, that shouldn't have been there lol